rca/navigation/templatetags/navigation_tags.py

```diff
@@ -45,6 +45,8 @@ def primarynav(context):
                 tertiary_links = []
                 if tertiary_pages:
                     for i in tertiary_pages:
+                        if not i["page"] and not i["url"]:
+                            continue
```

Comment:
**suggestion (non-blocking)** Is there any worth in logging a warning here, or would that information not feed back anywhere helpful?

Reply:
:+1: